XenAdmin/Alerts/Types/ClientUpdateAlert.cs

```diff
@@ -153,40 +157,30 @@ public static void DownloadSource(IWin32Window parent)
             // If no update no need to show where to save dialog
             var clientVersion = Updates.ClientVersions.FirstOrDefault();
 
-            DownloadSourceAction downloadSourceAction;
             if (clientVersion == null)
             {
-                // There is no XCupdates.xml so direct to website.
+                // There is no XCUpdates.xml so direct to website.
                 Program.OpenURL(InvisibleMessages.WEBSITE_DOWNLOADS);
-                return;
             }
             else
             {
                 string outputPathAndFileName;
                 using (var saveSourceDialog = new SaveFileDialog())
                 {
-                    saveSourceDialog.FileName = string.Format(Messages.SOURCE_FILE_NAME, BrandManager.BrandConsole, clientVersion.Version) + ".zip";
+                    saveSourceDialog.FileName = $"{BrandManager.BrandConsole}-v{clientVersion.Version}-source.zip";

```

Comment:
I think we should not localise this because it is the filename of the deliverable as we publish it on the website. If the users want, they can always change the name when they save it.

Reply:
Fair enough